[CI Failure]: Acceptance (curriculum-admin/create-edit-and-delete-classroom) test failed due to "Protocol error (Page.stopScreencast): Session closed"

### CI Test Type

Acceptance

### Which CI step failed?

Acceptance (curriculum-admin/create-edit-and-delete-classroom)

### Stacktrace

```shell
console.log
    LOG: User superAdm has the blog admin role!

      at Object.showMessage (utilities/common/show-message.ts:22:11)

  console.log
    LOG: User curriculumAdm has the curriculum admin role!

      at Object.showMessage (utilities/common/show-message.ts:22:11)

  console.log
    LOG: Page rendered fully.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: Subtopic Test Subtopic 1 is created.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: There are 0 classrooms present.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: Created Math classroom.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: There are 1 classrooms present.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: Updated Math classroom.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: Added Test Topic 1 topic to the Math classroom.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: Published Math classroom.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: The Math classroom has 1 topics.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: The Math classroom name is visible.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: Deleted Math classroom.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

  console.log
    LOG: There are 0 classrooms present.

      at Object.showMessage (utilities/common/show-message.ts:22:11)
          at runMicrotasks (<anonymous>)

FAIL core/tests/puppeteer-acceptance-tests/specs/curriculum-admin/create-edit-and-delete-classroom.spec.ts (317.502 s)

  ● Test suite failed to run

    Protocol error (Page.stopScreencast): Session closed. Most likely the page has been closed.

      at CDPSession.send (../../../node_modules/puppeteer/src/common/Connection.ts:286:9)
      at pageVideoStreamCollector.stopScreenCast (../../../node_modules/puppeteer-screen-recorder/src/lib/pageVideoStreamCollector.ts:81:26)
      at pageVideoStreamCollector.startSession (../../../node_modules/puppeteer-screen-recorder/src/lib/pageVideoStreamCollector.ts:89:16)
          at runMicrotasks (<anonymous>)
      at pageVideoStreamCollector.registerTabListener (../../../node_modules/puppeteer-screen-recorder/src/lib/pageVideoStreamCollector.ts:51:5)

  ● Test suite failed to run

    Protocol error (Page.stopScreencast): Session closed. Most likely the page has been closed.

      at CDPSession.send (../../../node_modules/puppeteer/src/common/Connection.ts:286:9)
      at pageVideoStreamCollector.stopScreenCast (../../../node_modules/puppeteer-screen-recorder/src/lib/pageVideoStreamCollector.ts:81:26)
      at pageVideoStreamCollector.startSession (../../../node_modules/puppeteer-screen-recorder/src/lib/pageVideoStreamCollector.ts:89:16)
          at runMicrotasks (<anonymous>)
      at pageVideoStreamCollector.registerTabListener (../../../node_modules/puppeteer-screen-recorder/src/lib/pageVideoStreamCollector.ts:51:5)

Test Suites: 1 failed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        317.567 s, estimated 364 s
Ran all test suites matching /\/home\/runner\/work\/oppia\/oppia\/core\/tests\/puppeteer-acceptance-tests\/specs\/curriculum-admin\/create-edit-and-delete-classroom/i.
Warning: 17 10:11:14 +0000] [9103] [WARNING] Worker with pid 9106 was terminated due to signal 15
Warning: 17 10:11:14 +0000] [9610] [WARNING] Worker with pid 9612 was terminated due to signal 15
Warning: 17 10:11:14 +0000] [9267] [WARNING] Worker with pid 9277 was terminated due to signal 15

i  emulators: Received SIGTERM for the first time. Starting a clean shutdown.
i  emulators: Please wait for a clean shutdown or send the SIGTERM signal again to stop right now.
i  Automatically exporting data using --export-on-exit "/home/runner/work/oppia/oppia/../firebase_emulator_cache" please wait for the export to finish...
⚠  Emulator UI has exited upon receiving signal: SIGTERM
i  Found running emulator hub for project dev-project-id at http://localhost:4400
i  Exporting data to: /home/runner/work/oppia/firebase_emulator_cache
i  emulators: Received export request. Exporting data to /home/runner/work/oppia/firebase_emulator_cache.
✔  emulators: Export complete.
✔  Export complete
i  emulators: Shutting down emulators.
i  ui: Stopping Emulator UI
⚠  ui: Error stopping Emulator UI
i  auth: Stopping Authentication Emulator
i  hub: Stopping emulator hub
i  logging: Stopping Logging Emulator
ERROR:root:Portserver failed to shut down after 10 seconds.
Starting new Portserver: python -m scripts.run_portserver --portserver_unix_socket_address /home/runner/work/oppia/oppia/portserver.socket
Starting new Redis Server: /home/runner/work/oppia/oppia/../oppia_tools/redis-cli-6.2.4/src/redis-server redis.conf
Starting new ElasticSearch Server: /home/runner/work/oppia/oppia/../oppia_tools/elasticsearch-7.17.0/bin/elasticsearch -q
Starting new Firebase Emulator: /home/runner/work/oppia/oppia/node_modules/firebase-tools/lib/bin/firebase.js emulators:start --only auth --project dev-project-id --config .firebase.json --export-on-exit /home/runner/work/oppia/oppia/../firebase_emulator_cache
Starting new Cloud Datastore Emulator: /home/runner/work/oppia/oppia_tools/google-cloud-sdk-500.0.0/google-cloud-sdk/bin/gcloud beta emulators datastore start --project dev-project-id --data-dir /home/runner/work/oppia/oppia/../cloud_datastore_emulator_cache --host-port localhost:8089 --consistency=1.0 --quiet --no-store-on-disk
Starting new GAE Development Server: /opt/hostedtoolcache/Python/3.9.20/x64/bin/python /home/runner/work/oppia/oppia_tools/google-cloud-sdk-500.0.0/google-cloud-sdk/bin/dev_appserver.py --host 0.0.0.0 --port 8181 --admin_host 0.0.0.0 --admin_port 8000 --enable_host_checking true --automatic_restart false --skip_sdk_update_check true --log_level error --dev_appserver_log_level error app.yaml
Starting new Acceptance Tests Server: /home/runner/work/oppia/oppia/node_modules/.bin/jest /home/runner/work/oppia/oppia/core/tests/puppeteer-acceptance-tests/specs/curriculum-admin/create-edit-and-delete-classroom --config=./core/tests/puppeteer-acceptance-tests/jest.config.js
Servers have come up.

Stopping Acceptance Tests Server(pid=9058)...
Stopping GAE Development Server(name="sh", pid=9004)...
Error: Process completed with exit code 1.
```

### Screenshots (if e2e tests)

_No response_

### Occurrences

Occured on - https://github.com/oppia/oppia/actions/runs/13895180166/job/38876411159?pr=21241

### Additional Information

_No response_

### Debugging document link

_No response_

--- a/core/tests/puppeteer-acceptance-tests/utilities/user/curriculum-admin.ts
+++ b/core/tests/puppeteer-acceptance-tests/utilities/user/curriculum-admin.ts
@@ -2445,6 +2445,7 @@ export class CurriculumAdmin extends TopicManager {
     await this.page.waitForSelector(openTopicDropdownButton);
 
     // Wait for the topic to appear in the classroom before adding prerequisites.
+    // Increased timeout to 60s because addTopicId makes an async API call that can take time.
     await this.page.waitForFunction(
       (
         topicBoxSelector: string,
@@ -2460,7 +2461,7 @@ export class CurriculumAdmin extends TopicManager {
         }
         return false;
       },
-      {},
+      {timeout: 60000},
       classroomTopicBoxSelector,
       classroomTopicNameSelector,
       topicName

